tenets/codelingo/effective-go/package-comment/codelingo.yaml

```diff
@@ -0,0 +1,25 @@
+tenets:
+  - name: package-comment
+    flows:
+      codelingo/docs:
+        name: Package comment
+        body: |
+          Every package should have a package comment, a block comment preceding the package clause. 
+          For multi-file packages, the package comment only needs to be present in one file, and any 
+          one will do. The package comment should introduce the package and provide information 
+          relevant to the package as a whole. It will appear first on the godoc page and should set up 
+          the detailed documentation that follows.
+      codelingo/review:
+        comment: |
+          Every package should have a package comment, a block comment preceding the package clause.
+          For multi-file packages, the package comment only needs to be present in one file, and any one will do.
+    query: |
+      import codelingo/ast/go
+
+      @review comment
+      go.file(depth = any):
```

Comment:
im a little concerned about removing the go.dir. if it's removed then you'll have to add a check to make sure that the files being compared live in the same repository. by putting (depth = any) onto the file fact you allow it to compare any two files. this might be the cause of so many hits.

alternatively, you might be able to add a js function to compare the directories by using the filename fact from each file

Reply:
I'm not sure what you mean here. How are we comparing two files? There's only a single file fact in the query.

Reply:
Just saw this sorry. The 2nd `go.file` is on line 26